astronomy-pipeline/astronomy_transform_functions.py

```diff
@@ -51,7 +53,29 @@ def get_body_mapping() -> dict:
     return b_mapping
 
 
-def clean_position_data(df: pd.DataFrame) -> pd.DataFrame:
+def get_constellation_mapping() -> dict:
```

Comment:
Good function for getting the new constellations mappings for the amended schema. Along with type hints and a descriptive doc string.